terraform/aptos-node/gcp/README.md

```diff
@@ -0,0 +1,180 @@
+# Using Terraform
+
+## Run on GCP
+This guide assumes you already have GCP account setup, and have created a new project for deploying Aptos node.
+
```

Comment:
Maybe add: See the section [Prerequisites](https://aptos.dev/tutorials/run-a-fullnode-on-gcp#prerequisites) for how to do this.

Reply:
GCP → GCP (Google Cloud Platform) 
(only in this first instance).